### PR TITLE
[3361][MIG] website_sale_wishlist_archive_cron: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3361](https://www.quartile.co/web#view_type=form&model=project.task&id=3361&active_id=3361&menu_id=)

### CODE REVIEW

File: website_sale_wishlist_archive_cron/models/product_wishlist.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class ProductWishlist(models.Model):
+    _inherit = "product.wishlist"
+
+    @api.model
+    def _process_wishlist_archive(self):
+        wishlist_records = self.search([("active", "=", True)])
+        for record in wishlist_records:
+            if not record.product_id.active:
+                record.active = False
```
Comment (yostashiro) [inline website_sale_wishlist_archive_cron/models/product_wishlist.py]:
```suggestion
        wishlist_records_to_archive = wishlist_records.filtered(lambda x: not x.product_id.active)
        wishlist_records_to_archive.write({"active": False})
```

**Review by yostashiro — CHANGES_REQUESTED** [review]
Rather minor suggestion.

**Comment by yostashiro** [comment]
@rinaldifirdaus Can you please double check the necessity of this module, according to the 15.0 functions (make sure that the same function does not exist).

**Comment by AungKoKoLin1997** [comment]
@rinaldifirdaus FYI, in odoo15, there is no function for auto archive for wishlist when archive the product but I didn’t check the cron job.

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
This is the functionality you get this module; the default Odoo15 does not have the same functionality.

**Pull request merged**